pypit/data/settings/settings.baseargflag

```diff
@@ -59,8 +59,8 @@ arc calibrate IDpixels []            # Manually set the pixels to be identified
 arc calibrate IDwaves []             # Manually set the corresponding ID wavelengths
 arc calibrate nfitpix  5             # Number of pixels to fit when deriving the centroid of the arc lines (an odd number is best)
 arc calibrate lamps None           # name of the ions used for the wavelength calibration
-arc calibrate method simple          # What method should be used to fit the individual arc lines (options are: fit, simple); fit is perhaps the most accurate; simple uses a polynomial fit (to the log of a gaussian), is the fastest and is reliable
-arc calibrate detection 6.0         # How significant should the arc line detections be (in units of a standard deviation)
+arc calibrate method arclines          # What method should be used to fit the individual arc lines (options are: fit, simple); fit is perhaps the most accurate; simple uses a polynomial fit (to the log of a gaussian), is the fastest and is reliable
+arc calibrate detection 1.0         # How significant should the arc line detections be (in units of a standard deviation)
```

Comment:
1.0 sounds aggressive..

Reply:
I've now removed the continuum algorithm and this gives a much better result all around. Therefore, I've changed this back to the original value, but at present, this variable doesn't do anything in the code.